build/tasks/install_jsdom.js

```diff
@@ -0,0 +1,29 @@
+module.exports = function( grunt ) {
+	grunt.registerTask( "jsdom", function() {
+		var done = this.async(),
+			pkg = grunt.config( "pkg" ),
+			version = pkg.jsdomVersions[
+
+				// Unfortunately, this is currently the only
+				// way to tell the difference between Node and iojs
+				/^v0/.test( process.version ) ? "node" : "iojs"
+			];
+
+		// Link npm so we can use the global one
```

Comment:
We could use command-line npm with another spawn. Now that it's just one npm command, I don't mind that as much.